### PR TITLE
feat(frontend): Add RTL support to the sidebar

### PR DESCRIPTION
**Pull request by Haskaris**
Issue #5289

Add the support RTL to the sidebar.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Comment by Haskaris** [comment]
From #dca1fa6
Z-index of wsContent is increased to go above the sidebar when fullscreen application.
Overwise we have to reduce the fullscreen application to let a margin-right (margin-inline-start) of the size of the sidebar.

File: frontend/src/css/Sidebar.styl [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
-    border-end-end-radius 15px
-    padding standardSpacing
     cursor pointer
     color offWhite
-    transition all sidebar-animate-speed
-    background-color var(--sidebarColor)
-    &.sidebarclose
-      left 0
+    display flex
+    min-width 30px
```
Comment (gbonaspetti) [inline frontend/src/css/Sidebar.styl:18]:
```suggestion
    min-width sidebarWidthReduced
```

File: frontend/src/container/Sidebar.jsx [hunk 2]
```diff
@@ -189,8 +190,8 @@ export class Sidebar extends React.Component {
       <div className='sidebar'>
         <div className={classnames('sidebar__expand', { sidebarclose: state.sidebarClose })} onClick={this.handleClickToggleSidebar}>
           {state.sidebarClose
-            ? <i className={classnames('fas fa-chevron-right')} title={props.t('See sidebar')} />
-            : <i className={classnames('fas fa-chevron-left')} title={props.t('Hide sidebar')} />}
+            ? <i className={classnames('fas', { 'fa-chevron-right': i18next.dir() === 'ltr' }, { 'fa-chevron-left': i18next.dir() === 'rtl' })} title={props.t('See sidebar')} />
+            : <i className={classnames('fas', { 'fa-chevron-left': i18next.dir() === 'ltr' }, { 'fa-chevron-right': i18next.dir() === 'rtl' })} title={props.t('Hide sidebar')} />}
```
Comment (gbonaspetti) [inline frontend/src/container/Sidebar.jsx:194]: With #5292 this will not be necessary

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: frontend/src/container/Sidebar.jsx (hunk 2 above)
Comment (Haskaris) [reply frontend/src/container/Sidebar.jsx]: Updated (removed) in ebd6fbd

**Review by gbonaspetti — APPROVED** [review]

**Comment by knodelle** [comment]
tested on branch

**Pull request merged**